Bottom: 4fe0c2f088cd86c22f8b7dffaacac90de17b1c32
Top:    fbcf43d1f5ebd0fa5f9d80e84466fbeb6dc21060
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2014-07-10 19:01:23 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
